aries_cloudagent/askar/store.py

```diff
@@ -49,9 +49,10 @@ def __init__(self, config: dict = None):
                 f"With key derivation method '{self.KEY_DERIVATION_RAW}',"
                 "key should also be provided"
             )
-        # self.rekey = config.get("rekey")
-        # self.rekey_derivation_method = config.get("rekey_derivation_method")
-
+        self.rekey = config.get("rekey")
+        self.rekey_derivation_method = (
+            config.get("rekey_derivation_method") or self.DEFAULT_KEY_DERIVATION
```

Comment:
How about implement the logic to set the default value when there is no setting in argparse.py?

Reply:
I think that's a good idea, but all the other configs used in the store use this pattern of loading with defaults here. I don't want to change it in this PR.